tasklauncher/spring-cloud-starter-stream-sink-tasklauncher-local/README.adoc

```diff
@@ -6,7 +6,15 @@ in the message that is received.
 
 Any message that does not contain a TaskLaunchRequest will log an error.
 
+== Using the TaskLauncher
+A tasklauncher-local is a sink that consumes a TaskLaunchRequest message, deploys and launches a task using the Spring
+Cloud Local Deployer.  The task will user the --uri from the message to determine what artifact to deploy as well as the
+properties and arguments to execute the task.  An example of this using Spring Cloud Data Flow would look like this:
 
+`stream create foo --definition "triggertask --uri=maven://org.springframework.cloud.task.app:timestamp-task:jar:1.0.0.BUILD-SNAPSHOT --fixed-delay=5 | tasklauncher-local"`
+
+Another example would be to use a processor to translate a message from another source into a TaskLaunchRequest that
+would be sent to the tasklauncher-local sink.
 //end::ref-doc[]
```

Comment:
Do we want to call out that for a system, you probably only need one of these (it can be clustered, but the idea of reusing the same sink for multiple scheduled tasks...)?